### DISCUSSION TITLE
Close Road Temporarily

### DISCUSSION THREAD

**Discussion by Scarlet-Dragon** [voxeltycoon/issues#343] [General]
A feature to "close" roads to future trucks for a temporary period of time. For example, have it look like a roadworks barricade, once it's put down, no further trucks will be routed that way (but existing trucks routed down there will continue) until the barricade has been removed. This would be handy during periods of road reconstruction like I want to turn a road into a bridge, so I provide a temporary diversion, get rid of the existing road, build the new bridge, reopen. Problem is, if there's a lot of trucks, it keeps saying "road occupied" for ages or I have to mess about with the trucks to be in a position to do the road works I want.

**Comment by StolleJay** [comment]
Like in openttd. I like it and would be nice to have it also here in VT :)
